### PR TITLE
fix(kvm): add encoding options to kvm page

### PR DESCRIPTION
**Pull request by chethanbellechalu-intel**

### CODE REVIEW

File: src/app/devices/kvm/kvm.component.html [hunk 1]
```diff
@@ -1,3 +1,16 @@
-<app-device-toolbar [deviceState]="deviceState" ></app-device-toolbar>
+<mat-toolbar>
+    <div fxFlex="25">
+        <mat-form-field appearance="fill" class="encoding-options">
```
Comment (rsdmike) [inline src/app/devices/kvm/kvm.component.html:3]: remove `appearance="fill"` and `class="encoding-options"`.  We aren't using this style anywhere on the site and want to stay consistent.

File: src/app/devices/kvm/kvm.component.scss [hunk 2]
```diff
@@ -8,6 +8,8 @@
     left: 50%;
 }
 .encoding-options{
```
Comment (rsdmike) [inline src/app/devices/kvm/kvm.component.scss:10]: remove added custom styles

File: src/app/devices/kvm/kvm.component.ts [hunk 3]
```diff
@@ -40,7 +40,7 @@ export class KvmComponent implements OnInit, AfterViewInit, OnDestroy {
   isPoweredOn: boolean = false
   isLoading: boolean = false
   deviceId: string = ''
-  selected: number = 2
+  selected: string = '1'
```
Comment (rsdmike) [inline src/app/devices/kvm/kvm.component.ts:43]: why can't we leave this as a number?

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: src/app/devices/kvm/kvm.component.html (hunk 1 above)
Comment (rsdmike) [reply src/app/devices/kvm/kvm.component.html:3]: Typicall we don't want to have a toolbar inside a toolbar. Place this toolbar after the `app-device-toolbar` with only the drop down, or perhaps ideally the user is prompted via a dialog box upon kvm connecting allowing them to choose.

File: src/app/devices/kvm/kvm.component.scss (hunk 2 above)
Comment (madhavilosetty-intel) [reply src/app/devices/kvm/kvm.component.scss]: done

File: src/app/devices/kvm/kvm.component.html (hunk 1 above)
Comment (madhavilosetty-intel) [reply src/app/devices/kvm/kvm.component.html]: done

**Review by rsdmike — APPROVED** [review]

**Pull request merged**